src/dedupe-hub/api/app/database/models/facility_claim.py

```
@@ -1,7 +1,8 @@
 from app.database.sqlalchemy import Base
-from sqlalchemy import Column, Integer
+from sqlalchemy import Column, Integer, String
 
 class FacilityClaim(Base):
     __tablename__ = 'api_facilityclaim'
 
-    id = Column(Integer, primary_key=True)
\ No newline at end of file
+    id = Column(Integer, primary_key=True)
+    origin_source = Column(String, nullable=False, default="os_hub")
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)